travis/deploy-daily-pypi-build.sh

```diff
@@ -12,4 +9,4 @@ echo "username:${TRAVIS_USERNAME}" >> ~/.pypirc
 echo "password:${TRAVIS_PASSWORD}" >> ~/.pypirc
 
 # Deploy daily pypi build
-.venv/bin/python setup.py egg_info --tag-date --tag-build=dev sdist bdist_wheel upload -r pypi
+python setup.py egg_info --tag-date --tag-build=dev sdist bdist_wheel upload -r pypi
```

Comment:
Are you sure that the project is build and ready to deploy ? (we use the container's python, I think that should pass but I'm not sure.)

Reply:
Yes, I tested by changeing the deploy configuration :-)